test/app-tap/module_api.c

```diff
@@ -2378,6 +2452,9 @@ luaopen_module_api(lua_State *L)
 		{"tuple_validate_def", test_tuple_validate_default},
 		{"tuple_validate_fmt", test_tuple_validate_formatted},
 		{"test_key_def_dup", test_key_def_dup},
+		{"session_check_admin", session_check_admin},
```

Comment:
You probably noticed, there are 38 lines above in all of which the functions have `test_` prefix. Please, conform. The test case functions must start with `test_`.

Reply:
They do not have prefix "test",  because they are called here "manually" here - https://github.com/picodata/tarantool/blob/module-api-add-box-session/test/app-tap/module_api.test.lua#L256-L261

while other tests are called here "automatically" - https://github.com/picodata/tarantool/blob/module-api-add-box-session/test/app-tap/module_api.test.lua#L242-L243

The initial idea was to reproduce `app-tap/module_api.test.lua`. It has some logic between tests like creation/dropping of tests and so on.